.github/workflows/release_droid_prepare_original_checksum.yml

```diff
@@ -22,8 +23,31 @@ jobs:
           key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
           restore-keys: |
             ${{ runner.os }}-maven-
+      - name: Enable Testconatiner reuse
+        run: echo 'testcontainers.reuse.enable=true' >> "$HOME"/.testcontainers.properties
+      - name: Create AWS test setup
+        run: |
+          echo "owner = \"$OWNER\"" > cloudSetup/terraform.tfvars
+          echo "deputy = \"$DEPUTY\"" >> cloudSetup/terraform.tfvars
+          terraform -chdir=cloudSetup/ init
+          terraform -chdir=cloudSetup/ apply -auto-approve
+        env:
+          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          OWNER: ${{ secrets.AWS_TAG_OWNER }}
+          DEPUTY: ${{ secrets.AWS_TAG_DEPUTY }}
       - name: Run tests and build with Maven
-        run: mvn -B clean verify --file pom.xml
+        run: mvn -B verify --file pom.xml -Dsonar.organization=exasol -Dsonar.host.url=https://sonarcloud.io -Dsonar.login=$SONAR_TOKEN
+        env:
+          GITHUB_OAUTH: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
```suggestion
```
You only need `GITHUB_TOKEN`.  We use it like this in OFT (https://github.com/itsallcode/openfasttrace/blob/develop/.github/workflows/build.yml#L52)